setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     aioconsole
     halo
     setuptools
+    hvac
```

Comment:
It would possibly be safer to pin this to v2 major version, to avoid breaking changes when they release a v3.

Reply:
Updated